lib/iris/tests/unit/fileformats/um/fast_load_structured_fields/test_BasicFieldCollation.py

```diff
@@ -222,7 +222,12 @@ def test(self):
             for test_tuple in test_date_tuples
         ]
         # Check all values are distinct.
-        self.assertEqual(len(test_date_ints), len(set(test_date_ints),))
+        self.assertEqual(
+            len(test_date_ints),
+            len(
+                set(test_date_ints),
+            ),
```

Comment:
This formatting looks really horrible, is less readable and seems gratuitous. Is there a configuration we can use to make Black less zealous?